File: src/main/java/org/intellij/sdk/language/jass/psi/JassTokenSets.java
package org.intellij.sdk.language.jass.psi;

import com.intellij.psi.tree.TokenSet;

public interface JassTokenSets {
    TokenSet IDENTIFIERS = TokenSet.create(JassTypes.KEY);

    TokenSet COMMENTS = TokenSet.create(JassTypes.COMMENT);
}

File: src/main/kotlin/raft/war/language/jass/psi/JassTokenSets.kt
package raft.war.language.jass.psi

import com.intellij.psi.tree.TokenSet
import raft.war.language.jass.psi.JassTypes

interface JassTokenSets {
    companion object {
        @JvmField
        val COMMENTS: TokenSet = TokenSet.create(raft.war.language.jass.psi.JassTypes.LINE_COMMENT)
    }
}
